src/components/Slider.jsx

```diff
@@ -0,0 +1,87 @@
+import { useRef, useState } from "react";
+import styled from "styled-components";
+
+import { activity } from "../libraries/slidercardsdata.json";
+import { SliderCard } from "./SliderCard";
+
+export const Slider = () => {
+  const [scrollPosition, setScrollPosition] = useState(0);
+
+  const containerRef = useRef();
+
+  const handleScroll = (scrollAmount) => {
```

Comment:
We thought this component really cool! You implemented useRef and useSate pretty well 💯 
// Sofie & Wen